### DISCUSSION TITLE
Image Displaying

### DISCUSSION THREAD

**Discussion by dev-dhull** [filamentphp/filament#5450] [Help]
Hii, my product image is uploading but it's not showing in filament listing
today i going live on server and we face that problem after live.

**Comment by m-pernkopf** [comment]
Do you get any errors in developer tools console or network? We had a similar problem with correct uploading, but not showing since the proper permissions have not been set on the resources folder, where the images have been stored (filamentuser is uploading the images, but www-data is trying to read the images). We fixed this by changing the images folder to the public folder.
